python/tank/errors.py

```diff
@@ -21,12 +21,21 @@ class TankError(Exception):
     pass
 
 
+class TankContextChangeNotAllowedError(TankError):
```

Comment:
This is super picky and really more asking you opinion really - just seeing the verb "allow" in the error  name makes me think it relates to permissions or something like that. Wondering if it could be called something like `ContextChangeNotPossibleError` or `ContextChangeNotSupportedError`. But - as I said - more a question than anything else...